ISISComputingGroup/EPICS-galil
# Alert when galil loses power

Added a PV which set a variable on a galil, every 5 minutes check that the SP and RB match, if they do, set a new value, if not then the Galil has power-cycled.
### Issue
https://github.com/ISISComputingGroup/IBEX/issues/4745
### To Test
Connect to a galil and set an initial value using `DMC01:pwrdet:SP` and then monitor it and `pwrdet_alert` once 5 minutes have passed `pwrdet_alert` should be 1, and a new value should automatically be set on `pwrdet:SP`. Next send the command RS to to the galil to reset it, this should be equivalent to power cycleing it. after 5 minutes `pwrdet_alert` should have value 0, and `pwrdet:SP` should not be updated to a new value
